jennifer_choi/class4/fizzbuzz.rb

```diff
@@ -0,0 +1,21 @@
+number = 0
+until number == 100
+	number += 1
+
+	if number%3 == 0
+		puts "Fizz"
+
+	elsif number%5 == 0
+		puts "Buzz"
+
+	elsif number%15 == 0
```

Comment:
Thanks Eddie! :)

*www.choijennifer.com* <http://www.choijennifer.com>

Jennifer Choi
917-756-4243

On Thu, Nov 19, 2015 at 7:13 PM, Eddie Washington <notifications@github.com>
wrote:

> In jennifer_choi/class4/fizzbuzz.rb
> <https://github.com/trivett/BEWD-NYC/pull/33#discussion_r45420794>:
>
> > @@ -0,0 +1,21 @@
> > +number = 0
> > +until number == 100
> > +	number += 1
> > +
> > +	if number%3 == 0
> > +		puts "Fizz"
> > +
> > +	elsif number%5 == 0
> > +		puts "Buzz"
> > +
> > +	elsif number%15 == 0
>
> nice fizz buzz! buuut, in the case that the number is 15 - the number will
> catch on the first conditional because 15 % 3 == 0 and the loop will
> print "Fizz" before it makes it down to the third conditional to print
> "FizzBuzz".
>
> Buuut, this is an easy fix! Since this ruby file is being interpreted line
> by line, we can just move the 3rd conditional to the top! like so:
>
> number = 0
> until number == 100
>   number += 1
>
>   if number % 15 == 0
>     puts "FizzBuzz"
>   elsif number % 3 == 0
>     puts "Fizz"
>   elsif number % 5 == 0
>     puts "Buzz"
>   end
>
> end
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/trivett/BEWD-NYC/pull/33/files#r45420794>.
>